model/src/main/kotlin/utils/FileArchiver.kt

```diff
@@ -81,7 +81,7 @@ class FileArchiver(
                 val relativePath = file.relativeTo(directory).invariantSeparatorsPath
 
                 if (tika.detect(file) != MimeTypes.PLAIN_TEXT) {
-                    logger.warn { "Not adding file '$relativePath' to archive because it is not a text file." }
+                    logger.debug { "Not adding file '$relativePath' to archive because it is not a text file." }
```

Comment:
I first did the same in #10553 but then removed that commit, because I think the reason to make this a warning was that the license file patterns should usually not match binary files. Let's hear another voice from @MarcelBochtler or @sschuberth.

Reply:
I think `debug` might be a little bit too hidden for this message, although other log statements in this area also use `debug` (it would be easier to see if this PR was rebased).

I'd be fine with keeping it at `warn`, but also lowering to `info` would be ok with me.

Reply:
@fviernau, could you please rebase to catch up with the latest changes in this code area?

Reply:
Rebased.